include/z64save.h

```diff
@@ -176,6 +176,32 @@ typedef enum TimerId {
 #define ENV_HAZARD_TEXT_TRIGGER_HOTROOM (1 << 0)
 #define ENV_HAZARD_TEXT_TRIGGER_UNDERWATER (1 << 1)
 
+typedef enum WorldMapArea {
+    /*  0 */ WORLD_MAP_AREA_HYRULE_FIELD,
+    /*  1 */ WORLD_MAP_AREA_KAKARIKO_VILLAGE,
+    /*  2 */ WORLD_MAP_AREA_GRAVEYARD,
+    /*  3 */ WORLD_MAP_AREA_ZORAS_RIVER,
+    /*  4 */ WORLD_MAP_AREA_KOKIRI_FOREST,
+    /*  5 */ WORLD_MAP_AREA_SACRED_FOREST_MEADOW,
+    /*  6 */ WORLD_MAP_AREA_LAKE_HYLIA,
+    /*  7 */ WORLD_MAP_AREA_ZORAS_DOMAIN,
+    /*  8 */ WORLD_MAP_AREA_ZORAS_FOUNTAIN,
+    /*  9 */ WORLD_MAP_AREA_GERUDO_VALLEY,
+    /* 10 */ WORLD_MAP_AREA_LOST_WOODS,
+    /* 11 */ WORLD_MAP_AREA_DESERT_COLOSSUS,
+    /* 12 */ WORLD_MAP_AREA_GERUDOS_FORTRESS,
+    /* 13 */ WORLD_MAP_AREA_HAUNTED_WASTELAND,
+    /* 14 */ WORLD_MAP_AREA_MARKET,
+    /* 15 */ WORLD_MAP_AREA_HYRULE_CASTLE,
+    /* 16 */ WORLD_MAP_AREA_DEATH_MOUNTAIN_TRAIL,
+    /* 17 */ WORLD_MAP_AREA_DEATH_MOUNTAIN_CRATER,
+    /* 18 */ WORLD_MAP_AREA_GORON_CITY,
+    /* 19 */ WORLD_MAP_AREA_LON_LON_RANCH,
+    /* 20 */ WORLD_MAP_AREA_20, // hakasitarelay and a lon lon ranch cutscene, gQuestionMarkPositionNameENGTex
+    /* 21 */ WORLD_MAP_AREA_GANONS_CASTLE,
```

Comment:
`WORLD_MAP_AREA_20` -> `WORLD_MAP_AREA_QUESTION_MARK` ?
I also had an aneurysm trying to read hakasitarelay cause i didnt realize it was referencing an og scene name.
Maybe `// Used by Windmill/Dampe Race and a Lon Lon Ranch Cutscene` ?

what cutscene btw? why does a cutscene have a unique map area, you cant pause?

Reply:
I don't remember which cutscene, and yeah it's probably a mistake as you can't pause in that scene layer
I can change to questionmark yeah

Reply:
fun fact: this enum is annoyingly close to matching the SPOT_XX order. The only difference is there is no SPOT_14 (14 maps to Market), and Lon Lon is SPOT_20 not 19.